prisma-fmt/src/get_config.rs

```diff
@@ -31,35 +32,36 @@ pub(crate) fn get_config(params: &str) -> Result<String, String> {
     get_config_impl(params)
         .map_err(|err| {
             json!({
-                "error": {
-                    "message": err.message,
-                    "error_code": err.error_code,
-                }
+                "message": err.message,
+                "error_code": err.error_code,
             })
             .to_string()
         })
         .map(|value| value.to_string())
 }
 
 fn get_config_impl(params: GetConfigParams) -> Result<serde_json::Value, GetConfigError> {
-    let mut config = match psl::parse_configuration(&params.prisma_schema) {
-        Ok(config) => config,
-        Err(err) => {
-            return Err(GetConfigError {
-                error_code: None,
-                message: err.to_pretty_string("schema.prisma", &params.prisma_schema),
-            })
+    let wrap_get_config_err = |errors: Diagnostics| -> GetConfigError {
+        use std::fmt::Write as _;
+
+        let mut full_error = errors.to_pretty_string("schema.prisma", &params.prisma_schema);
+        write!(full_error, "\nValidation Error Count: {}", errors.errors().len()).unwrap();
```

Comment:
The "Validation Error Count" was something appearing often in TS tests that I have figured was missing in the Wasm implementation. This PR fixes that.

Reply:
yep let's add this to make things match up exactly with the existing getConfig.